parsec/scheduling.c

```diff
@@ -168,15 +168,16 @@ int __parsec_execute( parsec_execution_stream_t* es,
             }
         }
 
+        parsec_hook_t *hook = tc->incarnations[chore_id].hook;
```

Comment:
the incarnations are supposed to be packed. How can we get a NULL hook here ?

Reply:
the incarnation exists because it has been created from a DTD task class, but the hook is not initialized because there is no device corresponding to the incarnation

Reply:
I cannot reproduce the error case I saw before, and I don't understand why it would have happened in the first place, so I just put an assert instead if it ever shows up again.